capstone-rs/src/test.rs

```diff
@@ -76,8 +78,10 @@ fn test_arm64_none() {
     assert!(cs.disasm_all(ARM_CODE, START_TEST_ADDR).unwrap().is_empty());
 }
 
+#[cfg(feature = "not_diet")]
 #[test]
 fn test_x86_names() {
+    use alloc::string::String;
```

Comment:
Let's put the import (protected by the cfg) at the top. We may need to use it in other function in the future.